ProductivitySuite/tests/test_productivity_suite_on_xeon.sh

```diff
@@ -0,0 +1,403 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set -xe
+echo "IMAGE_REPO=${IMAGE_REPO}"
+
+WORKPATH=$(dirname "$PWD")
+LOG_PATH="$WORKPATH/tests"
+ip_address=$(hostname -I | awk '{print $1}')
+
+function build_docker_images() {
+    cd $WORKPATH
```

Comment:
Please refer to the new structure, use a docker compose file instead of this part. 
https://github.com/opea-project/GenAIExamples/blob/main/SearchQnA/tests/test_searchqna_on_gaudi.sh

Reply:
https://github.com/opea-project/GenAIExamples/blob/main/SearchQnA/docker/docker_build_compose.yaml